server/start/cron-scheduler/scheduler-handler.ts

```diff
@@ -1,11 +1,117 @@
 import { jobs, SchedulerJob } from './index';
 import { configuredJobs } from './configured-jobs';
+import { log } from '../../lib/logger';
+import { getConfiguration } from '../../lib/get-configuration';
 import cron from 'node-cron';
+import { WAZUH_STATISTICS_TEMPLATE_NAME } from '../../../common/constants';
+import { statisticsTemplate } from '../../integration-files/statistics-template';
 
-
+const blueWazuh = '\u001b[34mwazuh\u001b[39m';
+const schedulerErrorLogColors = [blueWazuh, 'scheduler', 'error'];
 const schedulerJobs = [];
 
-export function jobSchedulerRun(context){
+/**
+* Wait until Kibana server is ready
+*/
+const checkKibanaStatus = async function (context) {
+  try {
+     log(
+       'scheduler-handler:checkKibanaStatus',
+       'Waiting for Kibana and Elasticsearch servers to be ready...',
+       'debug'
+     );
+ 
+    await checkElasticsearchServer(context);
+    await checkTemplate(context);
+    return;
+  } catch (error) {
+     log(
+       'scheduler-handler:checkKibanaStatus',
+       error.mesage ||error
+     );
+     try{
+       await delay(3000);
+       await checkKibanaStatus(context);
+     }catch(error){};
```

Comment:
This try-catch it's necessary?

Reply:
it's just a fuse, but the whole file needs to be refactored